### PR TITLE
Bankistapp

### PR DESCRIPTION
**Pull request by syedabdullahali**

### CODE REVIEW

**Comment by Rehan-Khan-Engineer** [comment]
We should always add a description and live link of  a PR! @syedabdullahali

File: style.css [hunk 1]
```diff
@@ -0,0 +1,654 @@
...
+    align-items: center;
+    flex-wrap: wrap;
+
+    height: inherit;
+}
+
+.form__input {
+    background-color: rgba(255, 255, 255, 0.51);
+    border: none;
+    border-radius: 10px;
```
Comment (Rehan-Khan-Engineer) [inline style.css:284]: border none but border radius has some value. Can you explain this @syedabdullahali

**Review by Rehan-Khan-Engineer — COMMENTED** [review]
Can you explain this project/repo @syedabdullahali ?

File: Script.js [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+
+const Button_Transfer = document.querySelector(".form_btn--transfer")
+const InputTransferTo = document.querySelector(".form_input--to")
+const InputAmount = document.querySelector(".form_input--amount")
+
+
+const button_close = document.querySelector(".form_btn--close")
+const input_close_user = document.querySelector(".form_input--user")
+const inpit_close_pin = document.querySelector(".form_input--pin")
+
```
Comment (Rehan-Khan-Engineer) [inline Script.js]: Is it  input_close_pin at line 110?

File: Script.js [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+
+const button_loan = document.querySelector(".form_btn--loan")
+const Input_Request_loan = document.querySelector(".form__input--loan-amount")
+const Button_Short = document.querySelector(".btn--sort")
+
+
+
+const LabelBalance = document.querySelector(".balnce_value")
+const labeldate = document.querySelector(".balance_data")
```
Comment (Rehan-Khan-Engineer) [inline Script.js]: const name should be in snakeCase here at line 120?

**Review by mujeeb100 — APPROVED** [review]
Nice!!!

**Pull request merged**